### DISCUSSION TITLE
请问通过shell脚本执行jar包，目前只能执行java8版本的，如何才能支持java11？

### DISCUSSION THREAD

**Discussion by 3813g00c** [apache/dolphinscheduler#8096] [Q&A] [answered]
我通过shell执行java jar包时，提示class版本过高，但是我们项目都是基于java11的，无法降级，可以通过升级dolphinscheduler版本或者其他方式解决吗

**Comment by YiuTerran** [comment, marked as answer]
你这显然是shell里默认java对应是8导致的，使用绝对路径的java11就可以了…

**Reply by 3813g00c** [reply]
是的，我们通过在镜像中安装java11 解决了
